packages/devtools_app/lib/src/shared/notifications.dart

```diff
@@ -39,6 +44,37 @@ class NotificationService {
   /// Pushes a notification [message].
   bool push(String message) => pushNotification(NotificationMessage(message));
 
+  /// Pushes an error notification with [errorMessage] as the text.
+  ///
+  /// Includes an action to report the error by opening the link to our issue
+  /// tracker.
+  bool pushError(String errorMessage) {
+    final reportErrorAction = NotificationAction(
+      'Report error',
+      () {
+        unawaited(
+          launchUrl(
+            devToolsExtensionPoints
+                .issueTrackerLink(
+                  issueTitle: 'Reporting error: $errorMessage',
+                )
+                .url,
+          ),
+        );
+      },
+    );
+    return pushNotification(
+      NotificationMessage(
+        errorMessage,
+        isError: true,
+        actions: [reportErrorAction],
+        // Double the duration so that the user has time to report the error:
+        duration: NotificationMessage.defaultDuration * 2,
```

Comment:
likely out of scope for this PR (since I don't think we have this for regular notifications?) but we should probably give the users a way to dismiss notifications. Especially long lasting ones like this one.

Reply:
SG! Will follow up with a PR to make notifications dismissable